docs/main/updating/5-0.md

```diff
@@ -37,15 +52,64 @@ androidxEspressoCoreVersion = '3.5.1'
 cordovaAndroidVersion = '10.1.1'
 ```
 
+### Update `google-services` in `build.gradle`
+
+```diff
+    dependencies {
+        classpath 'com.android.tools.build:gradle:7.4.1'
++       classpath 'com.google.gms:google-services:4.3.15'
+-       classpath 'com.google.gms:google-services:4.3.13'
+
+```
+
+### Update gradle plugin to 7.4.1 in `build.gradle`
+
+```diff
+    dependencies {
+-       classpath 'com.android.tools.build:gradle:7.2.1'
++       classpath 'com.android.tools.build:gradle:7.4.1'
+        classpath 'com.google.gms:google-services:4.3.15'   
+
+```
+
+### Update gradle wrapper to 7.5 in `gradle-wrapper.properties`
+
+```diff
+distributionBase=GRADLE_USER_HOME
+distributionPath=wrapper/dists
+- distributionUrl=https\://services.gradle.org/distributions/gradle-7.4.2-all.zip
++ distributionUrl=https\://services.gradle.org/distributions/gradle-7.5-all.zip
+zipStoreBase=GRADLE_USER_HOME
+zipStorePath=wrapper/dists
+```
+
+### Disable Jetifier in `gradle.properties`
+
+```diff 
+# Android operating system, and which are packaged with your app's APK
+# https://developer.android.com/topic/libraries/support-library/androidx-rn
+android.useAndroidX=true
+- # Automatically convert third-party libraries to use AndroidX
+- android.enableJetifier=true
+```
+
 ### Update androidScheme
 
 In Capacitor 6, `https` is going to be the default setting for `androidScheme` for __existing apps__.  To avoid data loss as a result of this change, you should set the scheme to `http` now even if it's the current default.
 
 ## Plugins
 
-The following plugin functions have been modified or removed. Update your code accordingly.
+The following plugin functionality has been modified or removed. Update your code accordingly.
 
 ### Device
 
 - `DeviceId.uuid` changed to `DeviceId.identifier`
-- On iOS 16+, `DeviceInfo.name` will return a generic device name unless you add the appropriate [entitlements](https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_developer_device-information_user-assigned-device-name).
\ No newline at end of file
+- On iOS 16+, `DeviceInfo.name` will return a generic device name unless you add the appropriate [entitlements](https://developer.apple.com/documentation/bundleresources/entitlements/com_apple_developer_device-information_user-assigned-device-name).
+
+### Push Notifications / Local Notifications
+
+- Starting in Android 13+, permissions are required to receive
```

Comment:
Looks like the sentence is incomplete? 

Also, I think it should be separate sections, since you receive push notifications, but you schedule local notifications

Reply:
Yeah, I was working on stuff in chunks so I just left that in as a placeholder to come back to it later.  Your point about making it separate sections is noted though.